### ISSUE TITLE
`copilot env init` ignores `AWS_PROFILE` variable  ?

### ISSUE THREAD

**Issue by rverpillot** [aws/copilot-cli#4049]
Why we need to use the flag `--profile` to specify a profile with `copilot env init` although I already setup the profile value in the `AWS_PROFILE` variable (https://aws.github.io/copilot-cli/docs/credentials) ? Is it a desired behavior ?

**Comment by dannyrandall** [comment]
Hey @rverpillot! Great question! 

Copilot can deploy environments in different AWS accounts or regions, so `env init` supports passing in a separate profile for a different account/region. You run `env init` with `AWS_PROFILE` set to _the account/region you deployed your app in_, but then Copilot creates an environment bootstrap stack with the `--profile` (or selected profile) account/region. Then, you continue using the application credentials, as Copilot deploys services in an environment with a role created on `env init`.

So, to sum up:
- `AWS_PROFILE` stays constant
- `--profile` is used if you want to set up an environment in a new account/region

**Comment by rverpillot** [comment]
Oh, very nice, I was not aware of this possibility. It's a great feature. I have to test it ASAP.
Thank for your response.
